packages/web3-eth/src/utils/get_revert_reason.ts

```diff
@@ -80,7 +85,8 @@ export async function getRevertReason<
 	web3Context: Web3Context<EthExecutionAPI>,
 	transaction: TransactionCall,
 	contractAbi?: ContractAbi,
-	returnFormat: ReturnFormat = DEFAULT_RETURN_FORMAT as ReturnFormat,
+	returnFormat: ReturnFormat = (web3Context.defaultReturnFormat ??
```

Comment:
as now we have default format coming from context so there isn't need of DEFAULT_RETURN_FORMAT form types package, and same of other places in this PR.